Services/CO.CDP.OrganisationInformation.Persistence/Migrations/20240919141551_UpdatePponSchemeName.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace CO.CDP.OrganisationInformation.Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class UpdatePponSchemeName : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("UPDATE identifiers SET scheme = 'GB-PPON' WHERE scheme = 'CDP-PPON';");
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
```

Comment:
Add migration down script, for just in case scenarios :)

Reply:
done